yalexs/activity.py

```diff
@@ -430,6 +443,16 @@ def operated_keypad(self):
         """Operation used keypad."""
         return self._info.get("keypad", self.action in KEYPAD_ACTIONS)
 
+    @cached_property
+    def operated_manual(self):
+        """Operation done manually using the knob."""
+        return self._info.get("manual", self.action in MANUAL_ACTIONS)
+
+    @cached_property
+    def operated_tag(self):
+        """Operation used rfid tag."""
+        return self._info.get("tag", self.action in TAG_ACTIONS)
```

Comment:
Looks like this is only hitting coverage from the `repr`, we should have an explicit test for both of these to make sure its true when its operated by the knob/tag